en/airframes_multicopter/qav-r-5-kiss-esc-racer.md

```diff
@@ -0,0 +1,166 @@
+
+<span></span>
+> **Tip** Put heat shrink on the cables before you start soldering! Heatshrinking the ESCs, the power module and the free floating uninsolated wire soldering joints after a successful function test will protect them from dirt, moisure and physical damage
+
+### Motors
+First I cut all three motor cables to directly fit when the ESCs are mounted on the arms shifted towards the center but still let enough slack to allow easy placement of the parts and not produce any tension on the cables. Then I soldered them in the order they come out of the motor to the output contacts of the ESCs which are oriented with the switching MOS-FETs facing updwards to get good air cooling during flight. Choosing this cable order resulted in all the motors spinning counter-clockwise in my tests and I switched where necessary the direction of rotation by bridging the dedicated [JP1 solder jumper](https://1.bp.blogspot.com/-JZoWC1LjLis/VtMP6XdU9AI/AAAAAAAAAiU/4dygNp0hpwc/s640/KISS-ESC-2-5S-24A-race-edition-32bit-brushless-motor-ctrl.jpg) to conform the [Quadrotor x configuration](../airframes/airframe_reference.html#quadrotor-x).
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_091317_s.jpg)
+
+### Power Module
+First I soldered the XT60 connector which comes with the frame to the labeled bettery side of the [ACSP5 power module](http://fs5.directupload.net/images/160304/a72l2mbz.jpg) that was shipped with the pixracer and added the elco capacitor delivered with the power module with the correct polarity to the same side.
+
+Now comes the tricky part. I soldered all four ESC voltage source + and - ports to the corresponding pad on the labeled ESC output side of the power module. Make sure to not have any cold solder joint here because the quad will not end up well with a loose connection in flight. Using the additional power distribution board of the frame would make the job a lot easier but also takes too much space on such a small frame...
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_093959_s.jpg)
+
+### Signal Cables
+
+I used thin cables with a standardized pin header connector which were cut in half for the ESC signal because this will allow easy plugging on the pixracer pins later on. Only the labeled `PWM` port on the [KISS ESCs](https://1.bp.blogspot.com/-0huvLXoOygM/VtMNAOGkE5I/AAAAAAAAAiA/eNNuuySFeRY/s640/KISS-ESC-2-5S-24A-race-edition-32bit-brushless-motor-ctrl.jpg) is necessary for flying. They will be connected to the correct motor signal output of the pixracer. The `TLM` port is for ESC telemetry and I soldered them on for future use as the needed protocol is not currently supported by PX4.
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_091354_s.jpg)
+
+I tested all ESC motor pairs and their rotation directions using a cheap PWM servo tester before proceeding further.
```

Comment:
I thought of that but then I thought you are totally screwed with this build if you can't solder and glue something... I can't cover the wheel. We should probably have a scetion in the future for such basic things or relate to other resources, I'm sure there are very nice instructions posted already somewhere.